roles/keystone-defaults/defaults/main.yml

```diff
@@ -10,6 +10,8 @@ keystone:
       - { name: mysql-python }
       - { name: functools32 }
       - { name: uwsgi }
+      - { name: python-ldap }
+      - { name: ldappool }
```

Comment:
Ah, crap. That's going to be a bit of a problem, will require new giftwrap packages.